### PR TITLE
Move GOVUK-Request-Id from varnish to nginx.

### PR DESCRIPTION
**Pull request by elliotcm**
DO NOT MERGE YET

This gives us tracing in our admin systems.

If we find that we need tracing back between varnish
and the router on our frontend systems we can add
a "GOVUK-Cache-Request-Id" or similar.

If anyone has any changes they'd like to make to the "unique enough" ID, please let me know.

I've confirmed via vagrant that `backend-lb-1` and `backend-1` will build and boot nginx happily, but had trouble trying to form convincing requests for it.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -126,6 +126,3 @@ DEPENDENCIES
   rspec-puppet
   sshkey (= 1.7.0)
   webmock (~> 1.20.0)
-
-BUNDLED WITH
-   1.10.5
```
Comment (rjw1) [inline Gemfile.lock]: this probably shouldnt be here. :)

Comment (elliotcm) [inline Gemfile.lock]: The removal shouldn't or the BUNDLED WITH shouldn't?  (it's largely been backed out from bundler)

**Comment by elliotcm** [comment]
Updated with config fix, shown working here: https://kibana.integration.publishing.service.gov.uk/kibana#dashboard/temp/AVQPwvUHAWArxCU_s3s8

**Comment by surminus** [comment]
:+1:

**Pull request merged**